src/pages/home/report/ReportActionsView.js

```diff
@@ -96,6 +96,7 @@ function ReportActionsView(props) {
 
     const isFocused = useIsFocused();
     const reportID = props.report.reportID;
+    const isNewestReportAction = lodashGet(props.reportActions[0], 'isNewestReportAction');
```

Comment:
```suggestion
    const hasNewestReportAction = lodashGet(props.reportActions[0], 'isNewestReportAction');
```

Because this is the list, it's not a report action. So this naming makes more sense

Reply:
agree